examples/flower_classifier/conda.yaml

```diff
@@ -0,0 +1,15 @@
+name: flower_classifier
+channels:
+  - defaults
+  - anaconda
+dependencies:
+  - python==3.6
+  - numpy==1.14.2
+  - keras==2.2.4
+  - pandas
+  - pip:
+    - tensorflow-gpu==1.10.0
+    - mlflow==0.8
```

Comment:
I think we should do mlflow>=0.8 so we don't have to change this for each release (or just do mlflow without specifying a version)